doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,598 @@
+  #.  Send telescope to flat-field position.
+  #.  Send dome to flat-field position.
+  #.  Put ``ATDomeTrajectory`` in ``ENABLED`` state.
+
+Make sure the system is ``ENABLED`` before running the task;
+
+.. code:: python
+
+    from lsst.ts.observatory.control.auxtel import ATCS, ATCSUsages
+
+    atcs = ATCS(intended_usage=ATCSUsages.PrepareForFlatfield)
+
+    await atcs.start_task
+
+    # put all ATCS components in ENABLED state
+    await atcs.enable()
+
+    # prepare ATCS for flat-field
+    await atcs.prepare_for_flatfield()
+
+To prepare the telescope for on-sky activities the class provides the task :py:meth:`prepare_for_onsky <lsst.ts.observatory.control.auxtel.ATCS.prepare_for_onsky>`, which can be used with :py:attr:`ATCSUsages.StartUp <lsst.ts.observatory.control.auxtel.ATCSUsages.StartUp>`.
+This method will perform the following tasks subsequently;
+
+  #.  Slew telescope to park position (in case telescope is in flat-field position or else).
+  #.  If primary mirror cover is open (e.g. for calibrations), close it. This is to ensure the mirror is protected when we start opening the dome, to avoid dust and particles from following in it.
+  #.  Move dome to oppose the setting Sun, to make sure no direct sunlight hits the inside of the dome and create thermal issues.
+  #.  Open dome slit.
+  #.  Once dome is open, open primary mirror cover and vent gates.
+  #.  Enable ``ATAOS`` corrections.
+
+In general, it is advised to make sure all components are in ``ENABLED`` state before running :py:meth:`prepare_for_onsky <lsst.ts.observatory.control.auxtel.ATCS.prepare_for_onsky>`, but the method also accepts a dictionary of ``settings`` and calls :py:meth:`enable <lsst.ts.observatory.control.RemoteGroup.enable>` at the beginning.
+
+.. code:: python
+
+    from lsst.ts.observatory.control.auxtel import ATCS, ATCSUsages
+
+    atcs = ATCS(intended_usage=ATCSUsages.StartUp)
+
+    await atcs.start_task
+
+    # prepare ATCS for flat-field
+    await atcs.prepare_for_onsky()
+
+Following up on what was shown in :ref:`user-guide-generic-csc-group-behavior`, the following is also a valid way of running :py:meth:`prepare_for_onsky <lsst.ts.observatory.control.auxtel.ATCS.prepare_for_onsky>`.
+
+Overriding the settings for a single component (e.g. ATAOS):
+
+.. code:: python
+
+    await atcs.prepare_for_onsky(settings={"ataos": "constant_hex"})
+
+Or Overriding the settings for all components:
+
+.. code:: python
+
+    await atcs.prepare_for_onsky(
+        settings={
+            "ataos": "current",
+            "atmcs": "",
+            "atptg": "",
+            "atpneumatics": "",
+            "athexapod": "current",
+            "atdome": "test.yaml",
+            "atdometrajectory": "",
+        }
+    )
+
+It is important to remember that, if the components are already enabled, they will be left in the ``ENABLED`` state and will not be re-cycled.
```

Comment:
Uh oh, so what happens if something is enabled already, then you send the enabled command with a different configuration than what it is enabled with? It throws an exception I hope?

Reply:
no... it is just ignored because the CSC is already enabled...